src/jit/unwindamd64.cpp

```diff
@@ -121,6 +161,8 @@ void Compiler::unwindPush(regNumber reg)
     unsigned int cbProlog = unwindGetCurrentOffset(func);
     noway_assert((BYTE)cbProlog == cbProlog);
     code->CodeOffset = (BYTE)cbProlog;
+
+    createCfiCode(func, cbProlog, OpAdjustCfaOffset, -1, 8);
```

Comment:
It looks like -1 is special for dwarfReg. You should add a `#define` like DWARF_REG_ILLEGAL or similar